schemas/context/opportunity-contactmap.example.1.json

```diff
@@ -0,0 +1,15 @@
+{
+    "12345": 
+      {
+        "xdm:roleId": "92312748749128",
+        "xdm:role": "finance",
+        "xdm:isPrimary":true,
+        "xdm:isDeleted":false
+      },
+      "12346":{
+        "xdm:roleId": "92312748749239",
+        "xdm:role": "sales",
+        "xdm:isPrimary":false,
+        "xdm:isDeleted":false
+      }
+  }
```

Comment:
Is there a reason the schema is in the "schemas/context" folder and the example is in "schemas/common". Can't we have the schema in the context like the "identityMap" ?